habitat_baselines/config/pointnav/ppo_pointnav_example.yaml

```diff
@@ -12,25 +12,26 @@ VIDEO_DIR: "video_dir"
 # To evaluate on all episodes, set this to -1
 TEST_EPISODE_COUNT: 2
 EVAL_CKPT_PATH_DIR: "data/new_checkpoints"
-NUM_PROCESSES: 1
+NUM_SIMULATORS: 1
 SENSORS: ["RGB_SENSOR", "DEPTH_SENSOR"]
```

Comment:
While we are at it, why do we override SENSORS on the DDPPO level. It's a bad idea as the sensorspec really should be defined by the task and this overrides those values.

Reply:
I agree.  This override is done because there isn't a way to change the habitat config from habitat baseline's config (nor is there a good way to do so with the way the configs are structured).

Reply:
There is a way to do with command line arguments, which I imagine would be the most common use case in practice.

Reply:
Yes and no.  I personally like having it in the config and the command line way is unintuitive because you have the TASK_CONFIG thing